modules/navigation/nav_utils.h

```diff
@@ -150,6 +150,25 @@ struct ClosestPointQueryResult {
 	RID owner;
 };
 
+enum PathfindingAlgorithm {
+	PATHFINDING_ALGORITHM_ASTAR = 0,
+};
+
+struct PathQueryParameters {
```

Comment:
Maybe these structs should be moved up into `NavigationServer3D` in order to use them to pass information up to that level, without having to depend on this module.  I noticed that the physics servers both do this with their queries.  It would mean adding a definition for a struct when we add GDExtension support, but that is something we can do.

Reply:
moved to `servers/navigation/navigation_utilities.h`